.github/workflows/linters.yml

```diff
@@ -0,0 +1,48 @@
+name: Linters
+
+on: pull_request
+
+env:
+  FORCE_COLOR: 1
+
+jobs:
```

Comment:
Please ensure that Linter is set up correctly for this repository. All Linter results should be green. AKA: Linter should report NO errors.